### DISCUSSION TITLE
Vim key navigation in file picker

### DISCUSSION THREAD

**Discussion by emirror-de** [helix-editor/helix#4646] [Ideas] [answered]
Hi all,

as far as I know, navigation in the file picker is currently only possible via arrow keys. In vim + telescope, it is possible to navigate also with `j` and `k` keys. The workflow is as follows:
1. Open file picker (`insert` mode)
2. Type search string
3. Press `ESC` (enter `normal` mode)
4. Navigate using `j` and `k`

Possible options in `normal` mode:
* Press `ESC` to exit the file picker
* Press `A` to go back to `insert` mode with cursor at the end of the search string

What do you think about this feature? Would it be interesting to implement this?

**Comment by the-mikedavis** [comment, marked as answer]
See https://github.com/helix-editor/helix/issues/2820

**Reply by emirror-de** [reply]
Oh, I somehow did not see it by myself while searching. Thank you! :rocket:

**Comment by Omnikar** [comment]
> as far as I know, navigation in the file picker is currently only possible via arrow keys.

You can also use tab and shift-tab, which is what I use personally.

**Reply by emirror-de** [reply]
Thats actually pretty nice! Definitely better than moving the hand to the arrow keys. Thanks for the hint!
